File: fizzbuzz.py
#!/usr/bin/python

for i in range(1,101):
        stdout = ''
        fizz = False
        buzz = False
        if i%3 == 0:
                stdout += 'Fizz'
                fizz = True

        if i%5 == 0:
                stdout += 'Buzz'
                buzz = True

        if fizz is False and buzz is False:
                stdout += str(i)

        print stdout

File: fizzbuzz.js
'use strict';

for (var i = 1, stdout, fizz, buzz; i <= 100; i++) {
	stdout = '';
	fizz = false;
	buzz = false;
	if (i%3 == 0) {
		fizz = true;
		stdout += 'Fizz';
	}
	if (i%5 == 0)  {
		buzz = true;
		stdout += 'Buzz';
	}
	if (!fizz && !buzz) {
		stdout += i;
	}
	console.log(stdout);
}
